var/spack/repos/builtin/packages/dire/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dire(Package):
+    """DIRE (short for dipole resummation) a C++ program for all-order
+    radiative corrections to scattering processes in high-energy particle
+    collisions."""
+
+    homepage = "http://dire.gitlab.io/"
+    url      = "http://dire.gitlab.io/Downloads/DIRE-2.004.tar.gz"
+    git      = "http://gitlab.com/dire/direforpythia"
+    list_url = "http://dire.gitlab.io/Downloads.html"
+
+    maintainer = ['mdiefent']
+
+    version('2.004', sha256='8cc1213b58fec744fdaa50834560a14b141de99efb2c3e3d3d47f3d6d84b179f')
+
+    depends_on('zlib')
+    depends_on('boost')
+    depends_on('lhapdf')
+    depends_on('hepmc')
+    depends_on('pythia8@8226:', when='@2.004:')
+
+    def install(self, spec, prefix):
+        configure_args = ['--prefix={0}'.format(prefix)]
+        configure_args.append(
+            '--with-pythia8={0}'.format(spec['pythia8'].prefix))
+        configure(*configure_args)
+        make()
+        # Open bug: https://gitlab.com/wdconinc/direforpythia/-/merge_requests/1
+        filter_file('-Wl,-rpath ', '-Wl,-rpath,', 'bin/dire-config')
```

Comment:
I wonder how this affects other compilers like NAG who use a different RPATH flag

Reply:
Unable to test at this point. I don't have access to a system with an existing NAG compiler. But I've requested a trial license, will test and report.

Reply:
I think I understand better now. NAG would require `-Wl,-Wl,,-rpath,` to pass this to the linker correctly. I'm not sure it is in scope to try to resolve this here though. dire-config is just a helper script for user code to allow e.g. ``gcc `dire-config --libs` source.cc``. dire-config does not have any way of knowing which compiler is used. Thoughts?

Reply:
If it's for C/C++ only, then I think the rpath flags would be the same since NAG is only a Fortran compiler. If you want to be careful, you can use `self.compiler.cc_rpath_arg`, but all the C/C++ compilers we currently support use the default flag.

Reply:
Implemented, tested, verified that the correct rpath lines appear in dire-config.